pkgs/development/tools/misc/rtss/default.nix

```
@@ -0,0 +1,23 @@
+{ stdenv, fetchFromGitHub, rustPlatform }:
```

```suggestion
{ lib, fetchFromGitHub, rustPlatform }:
```

stdenv can be replaced with lib.